vendor/QmXQEqXS3TDyhAFGBFeMadaBYJ2ismXXyQSn5i7HubcQUH/go-log/context.go

```diff
@@ -3,7 +3,7 @@ package log
 import (
 	"errors"
 
-	"github.com/ipfs/go-ipfs/Godeps/_workspace/src/golang.org/x/net/context"
+	"golang.org/x/net/context"
```

Comment:
thats '@whyrusleeping forgot to run godep save because we're still using godeps and it sucks' magic